src/resolution_functions/models/vision_paper.py

```diff
@@ -11,10 +20,67 @@
     from jaxtyping import Float
 
 
-
-
 @dataclass(init=True, repr=True, frozen=True, slots=True, kw_only=True)
 class VisionPaperModelData(ModelData):
+    """
+    Data for the `VisionPaperModel` model.
```

Comment:
We should probably give this model another name. This sounds like it came from one of the key papers describing the VISION instrument, which isn't really accurate.